test/hapi.stat.test.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+var assert = require('assert')
```

Comment:
Would it be better to use [code](https://github.com/hapijs/code) Instead?
(I can take care of this if you prefer)